Remove a confusing sentence from `about-code-owners.md` (#25692)
github/docs: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md

---
title: About code owners
intro: You can use a CODEOWNERS file to define individuals or teams that are responsible for code in a repository.

> permissions: People with write permissions for the repository can create or edit the CODEOWNERS file and can be listed as code owners in it. People with admin or owner permissions can require that pull requests by approved by code owners before they can be merged.

Reviewer: ```suggestion
permissions: 'People with write permissions for the repository can create or edit the CODEOWNERS file and be listed as code owners. People with admin or owner permissions can require that pull requests have to be approved by code owners before they can be merged.'
```

topics:
  - Repositories
---
The people you choose as code owners must have write permissions for the repository. When the code owner is a team, that team must be visible and it must have write permissions, even if all the individual members of the team already have write permissions directly, through organization membership, or through another team membership.

> ## About code owners

Reviewer 2: ```suggestion
## About code owners
```
